applications/NXmx.nxdl.xml

```diff
@@ -413,42 +546,38 @@
 					<field name="pixel_mask_applied" type="NX_BOOLEAN" minOccurs="0">
 						<doc>
 							True when the pixel mask correction has been applied in the
-							electronics, false otherwise.
+							electronics, false otherwise.  
 						</doc>
 					</field>
 
-					<field name="pixel_mask" type="NX_INT" minOccurs="0">
+					<field name="pixel_mask" type="NX_INT" recommended="true">
```

Comment:
Doc implies type should be NX_INT32

Reply:
I think NX_INT is adequate, but if somebody feels strongly they should change ii

Reply:
So if bit depth < 32 implies missing fields are false/unset?